src/order/complete_well_founded.lean

```diff
@@ -109,9 +188,14 @@ lemma is_sup_closed_compact_iff_well_founded :
   is_sup_closed_compact α ↔ well_founded ((>) : α → α → Prop) :=
 (well_founded_characterisations α).out 2 0
 
+lemma is_Sup_finite_compact_iff_all_elements_compact :
```

Comment:
Given that we essentially prove this above (via `all_elements_compact.is_Sup_finite_compact` and `is_Sup_finite_compact.all_elements_compact`) I think it might be better just to state and prove this above and not go via the `tfae` machinery.

Reply:
Yes, since this is already an iff between 2 & 4, it seems like a good idea to just establish the iff above; I'll do that.